cms/templates/settings.html

```diff
@@ -343,6 +343,33 @@ <h2 class="title-2">${_('Course Details')}</h2>
                     <button type="button" class="action action-upload-image" id="upload-web-science-image">${_("Upload Web Science Course Card Image")}</button>
                   </div>
                 </li>
+                <li class="field image" id="field-web-science-main-image">
+                  <label for="course-web-science-url">${_("Web Science course main Image")}</label>
```

Comment:
the same question about translates. And maybe better to use `platform_name` instead of `Web Science`?

Reply:
I'll change `Web Science` to `platform name`

Reply:
done